aidial_adapter_bedrock/utils/env.py

```diff
@@ -13,3 +15,18 @@ def get_env(name: str, err_msg: Optional[str] = None) -> str:
 
 def get_env_bool(name: str, default: bool = False) -> bool:
     return os.getenv(name, str(default)).lower() == "true"
+
+
+def get_aws_default_region() -> str:
+    region = os.getenv("DEFAULT_REGION")
+    if region is not None:
+        log.warning(
+            "DEFAULT_REGION env variable is deprecated. Use AWS_DEFAULT_REGION instead."
+        )
+        return region
+
+    region = os.getenv("AWS_DEFAULT_REGION")
```

Comment:
Should this one have higher priority?

Reply:
`DEFAULT_REGION` has a higher priority, because otherwise it will break the case when both `DEFAULT_REGION` and `AWS_DEFAULT_REGION` are set and not equal to each other.